widgetastic_manageiq/__init__.py

```diff
@@ -2573,14 +2573,19 @@ def _prepare_event(self, evt, category):
         node = document_fromstring(evt)
         # lxml doesn't replace <br> with \n in this case. so this has to be done by us
         for br in node.xpath("*//br"):
-            br.tail = "\n" + br.tail if br.tail else "\n"
+            br.tail = "\n" + br.tail.rstrip() if br.tail and br.tail.rstrip() else "\n"
```

Comment:
@john-dupuy I don't think we need `br.tail and br.tail.rstrip()` only `br.tail.rstrip()` check is sufficient. what do you think?

Reply:
@digitronik Are you referring to the `if br.tail and br.tail.rstrip()` line? The problem is that `br.tail` is either a string or `None` and if we just have `if br.tail.rstrip()` we are trying to call a method that doesn't exist for a `None` object.